int32                   loop
geometry_msgs/Pose[]    pose_list
string[]                name_list
